libraries/iio/iio.c

```diff
@@ -0,0 +1,843 @@
+/***************************************************************************//**
```

Comment:
Let's use `git mv` or something that keeps track of history when renaming (moving) files around.

Reply:
https://github.com/analogdevicesinc/no-OS/pull/673 how about this?

Reply:
Solved here: https://github.com/analogdevicesinc/no-OS/pull/657